### PR TITLE
feat: update to angular 2.0.0-alpha.32. Also simplified typings

### PR DESCRIPTION
**Pull request by NathanWalker**
Simplified typings to be consistent approach across most all seed projects now in managing custom typings vs. tsd installed definitions. Suggestion by @gdi2290

### CODE REVIEW

**Comment by gdi2290** [comment]
:+1:

File: app/index.html [hunk 1]
```diff
@@ -3,8 +3,10 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <base href="<%= APP_BASE %>">
   <title>My Angular 2 App</title>
+  <base href="<%= APP_BASE %>">
+  <!-- ignore: Aplha 32 router fix -->
+  <script>baseElement = document.querySelector('base');baseElement.attr = baseElement.getAttribute;</script>
```
Comment (NathanWalker) [inline app/index.html]: This is also what @gdi2290 did as well:
https://github.com/angular-class/angular2-webpack-starter/blob/0122d866cfcffb76150aa3047e6f9adb1959733d/src/public/index.html#L15-16

**Comment by NathanWalker** [comment]
Additionally I issued a PR to DefinitelyTyped for systemjs based on what you guys had mocked out:
https://github.com/borisyankov/DefinitelyTyped/pull/5043
:)

**Comment by ludohenin** [comment]
@NathanWalker I have a lot of typing error when building the app. Do you also have some ?

**Comment by ludohenin** [comment]
A lot of `Duplicate identifier` actually and few `not found` ones.

**Comment by ludohenin** [comment]
@NathanWalker Found the duplicates, this one `/tsd_typings/custom.system.d.ts` is to be removed.

**Comment by NathanWalker** [comment]
Probably best to remove the `typings` folder in root altogether if one was still there.
Then `tsd reinstall` to get them fresh.

Custom type definitions are to go `only` in `app/typings`... everything in `tsd_typings` should be straight from the `tsd install/reinstall` command based off `tsd.json` :+1:

**Comment by ludohenin** [comment]
right

**Comment by NathanWalker** [comment]
I'll clean that up... 1 min, I'll commit a fix.

**Comment by ludohenin** [comment]
but the compiler still can't find abs ref `<reference path="/app/typings/_custom.d.ts" />` in app.ts, home.ts...

**Comment by NathanWalker** [comment]
The above commit https://github.com/NathanWalker/angular2-seed/commit/eb1e3d1c2d7bd7655602c0e9e4b869fb3fe0ce36 should fix it.

**Comment by ludohenin** [comment]
Looks good :-)
`experimentalDecorators` set to `true` in `tsconfig.js` and that should be perfect.
thanks,

**Comment by NathanWalker** [comment]
Ah yes, good catch! Thanks @ludohenin I got the absolute references cleaned up too, all is solid :+1:

**Comment by ludohenin** [comment]
@mgechev I think it is ready to merge

**Comment by mgechev** [comment]
Yep, just too a look. Looks great!

**Pull request merged**